Packaging/nix/devilutionx.metainfo.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.diasurgical.DevilutionX</id>
+  
+  <name>DevilutionX</name>
```

Comment:
Let's just localize this file inline? Seems simpler than doing it via gettext and xml merging.

Reply:
I don't know what you're talking about, but whatever works :)